src/core/entry/cards/Movie.tsx

```diff
@@ -57,7 +58,9 @@ const MovieCard: FC<Props> = ({ entry, className, children }) => {
       >
         <div css={titleArea}>
           <Typography css={selectableType} variant="h5" component="h2" color="textPrimary">
-            {movieName} ({movieYear})
+            {movieName}
+            {originalname !== movieName && ' - '}
+            {originalname !== movieName && originalname} ({movieYear})
```

Comment:
Nit: check to make sure there is an original name and just simplifies it a little bit. 
```suggestion
            {originalName && originalName !== movieName && ` - ${originalName}`}
           ({movieYear})
```